frontend/src/components/place/AddPlace.vue

```diff
@@ -637,7 +637,9 @@ export default {
 
         LocationService.getUserLocationData()
             .then(coordinates => {
-                LocationService.getCityByLocation(mapSettingsService.getMapboxToken(), coordinates.lng, coordinates.lat)
+                let params = coordinates.lng + ',' + coordinates.lat;
+                let mapboxCitiesApiUrl = mapSettingsService.getMapboxCitiesApiUrl(mapSettingsService.getMapboxToken(), params);
+                LocationService.getCityList(mapboxCitiesApiUrl)
```

Comment:
getCityList(params)
you can call mapSettingsService.getMapboxCitiesApiUrl(params) inside getCityList